ex2/ex2.c

```diff
@@ -9,6 +9,19 @@
 int main(void)
 {
     // Your code here 
+    FILE * fp;
+    // Creates an empty file for both reading and writing.
+	fp = fopen ("text.txt", "w+");
+	int y = fork();
+
+	if (y == 0){
+		// Parent should write first
+		fprintf(fp, "%s %s %s %d", "We", "are", "in", 2019);
+	} else{
+		fprintf(fp, "%s %s %s %d", "We", "are", "in", 2020);
+	}
```

Comment:
You are correct in saying the parent should write first but is the _child_ or _parent_ printing the year **2019**? 

Also it is not in a very pretty format when you view it as a text file. Would you mind prettying it up a bit tomorrow?